examples/ios/TextSort.xcodeproj/project.pbxproj

```diff
@@ -428,6 +428,7 @@
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "TextSort/TextSort-Prefix.pch";
 				INFOPLIST_FILE = "TextSort/TextSort-Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
any reason for requiring 12.0?

Reply:
Latest XCode won't compile it without a min deployment target.
My reasoning was that iOS 12 is new enough to be supported by any modern xCode and old enough so no apple supported xCode doesn't support it. (you have to go back to xcode 8 or 9 which are out of the map)
So I thought this was likely the most compatible one. However, open to suggestions.

Reply:
I think that min version should be iOS 11.

Reply:
Cool 11 is good.